freqtrade/freqai/data_kitchen.py

```diff
@@ -780,6 +780,9 @@ def compute_inlier_metric(self, set_='train') -> None:
         into previous timepoints.
         """
 
+        def normalise(dataframe: DataFrame) -> DataFrame:
+            return (dataframe - dataframe.min()) / (dataframe.max() - dataframe.min())
```

Comment:
If this function were used in a regular `populate_indicators()` method - it'd be a lookahead bias because it's using the full dataframe ...

Assmuing you considered this - please add a docstring here - which should clearly state why/when it's acceptable to use this.